login.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Linkbrary</title>
+  <link rel="stylesheet" href="css/login.css"><link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Montserrat+Alternates:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+  <link
+  href="https://fonts.googleapis.com/css2?family=Noto+Sans+KR:wght@400;700&family=Poppins:wght@400;700&display=swap"
+  rel="stylesheet"
+>
+</head>
+<body>
+  <div class="container">
+    <div class="login-top">
+      <a class="btn-home" href="./index.html">Linkbrary</a><br><br>
+      회원이 아니신가요? <a class="signUp" href="./signUp.html">회원 가입하기</a>
+    </div>
+    <div class="login-mid">
+      <form>
+        <label name="email">이메일<br>
+          <input type="email" name="email" id="input-email"><br>
+        </label>
+        <label name="password"> 비밀번호<br>
+          <input type="password" name="password" id="input-password">
+        </label>
+        <button class="btn login" style="cursor: pointer;">로그인</button>
+      </form>
+    </div>
+    <div class="login-btm">
+      <div class="snsLogin-container">
+        소셜 로그인
+        <div class="btn-snsLogin">
+          <a href="#"><img class="google-login" src="./img/icn_goolge.png"></a>
+          <a href="#"><img class="kakao-login" src="./img/icn_kakao.png"></a> 
+        </div>
+      </div>
+    </div>
+  </div>
```

Comment:
@wjy308 

이 부분도 마크업이 조금 더 명확하면 좋겠어요! 
container라는 클래스는 UI 개발에서 특정 목적으로 활용되는 경우가 많습니다. 
또한 이 경우, 로그인 폼을 그리기 위해 필요한 div이죠? 
따라서 container라는 이름은 조금 적합하지 못하다고 생각이 되어요. 

이런식으로 개선해보면 어떨까요? 

```suggestion
<div class="login-form">
  <header class="login-form__header">
    <a ...
    <p ... <a>회원 가입하기 ...
  </ header>
  <form class="login-form__body">
     <div class="form-group">
        <label>
        <input>
        <p>
      </ div>
...
```